pkg/clustermanager/reconcile_logicalcluster.go

```diff
@@ -53,9 +53,45 @@ func (c *controller) reconcileLogicalCluster(key string) error {
 	return nil
 }
 
-// TODO: the processAddLC needs to check whether the LogicalCluster has already
-// been added and processed, or whether this is actually an Update.
-func (c *controller) processAddLC(newCluster *lcv1alpha1.LogicalCluster) error {
+// processAddLC: process an added LC object
+func (c *controller) processAddLC(logicalCluster *lcv1alpha1.LogicalCluster) error {
+	logger := klog.FromContext(c.context)
+	var err error
+
+	switch logicalCluster.Status.Phase {
+	case lcv1alpha1.LogicalClusterPhaseEmpty:
+		// Either the client created a new logical cluster object and expects us to
+		// manage/create a physical cluster, or the discovery identified a new
+		// physical cluster.
```

Comment:
If the discovery found a new cluster it sets it to READY

Reply:
Correct.  I have that spelled out in the TODO for processNewLC.

Reply:
I mean you don't go here as a result of discovery. so please fix the comment.